bin/mailsync

```diff
@@ -6,6 +6,7 @@
 pgrep -u "${USER:=$LOGNAME}" >/dev/null || { echo "$USER not logged in; sync will not run."; exit ;}
 # Run only if not already running in other instance
 pgrep -x mbsync >/dev/null && { echo "mbsync is already running." ; exit ;}
+while pgrep -x ${LOCK:-slock} >/dev/null; do sleep 10; done
```

Comment:
`pgrep -x  ${LOCK:-slock} >/dev/null && { echo "user session is locked, assume passwords are locked aswell." ; exit ;}`
Could be an alternative solution if you want to kill `mailsync` when the screen is locked and let it be run triggered by cron/systemd timer at a later moment.